### PR TITLE
feat: add ListById function and test

### LINKED ISSUE #95: Add `GetRuleById()` function
I recently used your package to manage `iptables` forwards (thanks!).
I would introduce a new function:
```golang
func GetRuleById(table string, chain string, id int) error {
    ...
}
```
This allow you to retrieve a specific rule by passing its number identifier.
The number is retrievable using the following (example) command:
```bash
$ sudo iptables -t nat -L PREROUTING -n --line-number
Chain PREROUTING (policy ACCEPT)
num  target     prot opt source               destination         
1    DNAT       tcp  --  0.0.0.0/0            0.0.0.0/0            tcp dpt:1242 to:127.0.0.1:29
2    DNAT       tcp  --  0.0.0.0/0            0.0.0.0/0            tcp dpt:1242 to:127.0.0.1:22
3    DNAT       tcp  --  0.0.0.0/0            0.0.0.0/0            tcp dpt:1242 to:127.0.0.1:25
```
This is really useful when you need information or need to delete a specific rule.

### PR DESCRIPTION
**Pull request by alegrey91**
Fix #95

### CODE REVIEW

File: iptables/iptables.go [hunk 1]
```diff
@@ -220,6 +220,20 @@ func (ipt *IPTables) DeleteIfExists(table, chain string, rulespec ...string) err
 	return err
 }
 
+// GetRuleById return rule specifing its Id
+func (ipt *IPTables) GetRuleById(table, chain string, ruleId int) (string, error) {
```
Comment (squeed) [inline iptables/iptables.go:224]: `iptables -S` takes an optional `rulenum`, e.g.

```
$ iptables -S FORWARD 1
-A FORWARD -j DOCKER-USER
```

Given that iptables is 1-indexed, so the library should match this.

**Review by squeed — CHANGES_REQUESTED** [review]
ruleId should be one-indexed to match the `iptables` binary.

**Comment by alegrey91** [comment]
Hi @squeed, thanks for your reply.
So, you mean something like this?
```golang
// List rules in specified table/chain
func (ipt *IPTables) ListById(table, chain string, Id int) (string, error) {
	args := []string{"-t", table, "-S", chain, strconv.Itoa(Id)}
	rule, err := ipt.executeList(args)
	if err != nil {
		return "", err
	}
	return rule[0], nil
}
```

**Comment by alegrey91** [comment]
I think could be more appropriate the name `ListById()` for this function, right?

File: iptables/iptables.go [hunk 2]
```diff
@@ -220,6 +220,16 @@ func (ipt *IPTables) DeleteIfExists(table, chain string, rulespec ...string) err
 	return err
 }
 
+// List rules in specified table/chain
+func (ipt *IPTables) ListById(table, chain string, Id int) (string, error) {
```
Comment (squeed) [inline iptables/iptables.go:224]: This looks good. One minor nit: `Id` should be lower-cased. Once that's fixed we can merge :-)

Comment (alegrey91) [reply iptables/iptables.go]: Done

**Review by squeed — APPROVED** [review]

**Pull request merged**